src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/page.tsx

```diff
@@ -13,11 +13,12 @@ import { WelcomeToPremiumView } from "./WelcomeToPremiumView";
 import { getSubscriberEmails } from "../../../../../../../../functions/server/getSubscriberEmails";
 import { StepDeterminationData } from "../../../../../../../../functions/server/getRelevantGuidedSteps";
 import { getCountryCode } from "../../../../../../../../functions/server/getCountryCode";
+import { activateAndOptoutProfile } from "../../../../../../../../functions/server/onerep";
 
 export default async function WelcomeToPremiumPage() {
   const session = await getServerSession(authOptions);
 
-  // Ensure user is logged in
+  // Ensure user is logged in and a subscriber
```

Comment:
So the term `subscriber` pre-dates the premium feature, I'm worried this is going to cause confusion... if they are logged in then they get a `subscriber.id`, not the fault of your comment but I'm worried that someone might think this means the user subscribed to premium.